### PR TITLE
mvn clean on dev-clean profile won't remove node_modules

### PR DESCRIPTION
**Pull request by sumengwang**

### CODE REVIEW

**Review by sechegaray — APPROVED** [review]

**Review by mrahanjam — APPROVED** [review]

**Review by tivv — CHANGES_REQUESTED** [review]
Please use a profile. By default `clean` should clean everything

**Review by de-lan — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -660,5 +658,28 @@
...
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-clean-plugin</artifactId>
+            <version>2.6.1</version>
+            <configuration>
+              <excludeDefaultDirectories>true</excludeDefaultDirectories>
+              <filesets>
+                <fileset>
+                  <directory>node_modules</directory>
+                  <excludes>
+                    <exclude>**</exclude>
```
Comment (tivv) [inline pom.xml]: Hm Are you sure it deletes anything with this settings? You excluded the defaults and I don't see any includes

Comment (sumengwang) [reply pom.xml]: You are right. I noticed that the clean without profile will delete the node_modules folder. So In the dev-clean profile, I removed the part that explicitly deletes node_modules

Comment (tivv) [reply pom.xml]: But are you sure it deletes other stuff? E.g. if target directory is present, does it delete it? Looking at the setting I don't see this settings deleting anything.

Comment (sumengwang) [reply pom.xml]: Yea I tested it. It will delete target/

**Review by tivv — APPROVED** [review]

**Pull request merged**